### PR TITLE
Fixes for getenv, still it doesn't work. pdksh added for test purposes.

### PR DESCRIPTION
**Pull request by elbing**

### CODE REVIEW

File: cmd/pdksh/Makefile [hunk 1]
```diff
@@ -0,0 +1,19 @@
+include ../../Makefile.inc
```
Comment (rminnich) [inline cmd/pdksh/Makefile]: Why not have $(APEX)/$(OS).inc?

**Comment by rminnich** [comment]
How about we clean up the makefile situation first?

**Comment by rminnich** [comment]
yeah, if you can yank the Makefile.inc from this PR, and approve #8, things should be much better.

**Comment by elbing** [comment]
Rebased.

**Comment by staalmannen** [comment]
how do you feel about mksh instead of pdksh? Build.sh works out of the box on Plan9/APE. mksh is actively developed. The only issue might be 64bit if it still is LLP64 on Harvey.

**Pull request closed without merging**